.travis.yml

```diff
@@ -6,16 +6,11 @@ notifications:
     on_failure: change
 
 php:
-  - 5.3
   - 5.6
 
 env:
   - WP_VERSION=latest WP_MULTISITE=0
-
-matrix:
```

Comment:
I think we still need the `matrix` to be able to test multsite + non-multisite separately.

Reply:
Should we be testing multisite on all PHP versions or only some?

The travis configuration before this commit was [only running multisite on PHP 5.3](https://travis-ci.org/Automattic/WPCOM-Legacy-Redirector/builds/282731430), while this PR [expands it to include PHP 5.6](https://travis-ci.org/Automattic/WPCOM-Legacy-Redirector/builds/282540193?utm_source=github_status&utm_medium=notification) (examples linked)

Reply:
I think it probably makes sense to test on all.

Reply:
as best as I understand, Travis will build a matrix using the default configuration set (in this case, all combinations of the `php` and `env` declarations), and the `matrix` declaration can be used to override that using `include` and/or `exclude`.

The documentation on this isn't super clear, but it's available [here](https://docs.travis-ci.com/user/customizing-the-build#Build-Matrix)

However, [Liveblog](https://github.com/Automattic/liveblog/blob/master/.travis.yml) & [Jetpack](https://github.com/Automattic/jetpack/blob/master/.travis.yml) (as examples) both avoid setting a default configuration in favor of defining an include matrix. Happy to switch this over if that's the preferred pattern.

Reply:
Oh, actually, I misunderstood. I think the approach we have here seems fine and works well :)